packages/astro/src/runtime/client/dev-overlay/plugins/audit/index.ts

```diff
@@ -198,6 +198,18 @@ export default {
 			const rect = originalElement.getBoundingClientRect();
 			const highlight = createHighlight(rect, 'warning');
 			const tooltip = buildAuditTooltip(rule, originalElement);
+
+			// Set the highlight/tooltip as being fixed position the highlighted element
+			// is fixed. We do this so that we don't mistakenly take scroll position
+			// into account when setting the tooltip/highlight positioning.
+			// 
+			// We only do this once due to how expensive computed styles are to calculate, 
+			// and are unlikely to change. If that turns out to be wrong, reconsider this.
+			const { isFixed } = getElementsPositionInDocument(originalElement);
```

Comment:
this logic was present in the xray logic, so felt like it made sense to add here as well